### PR TITLE
Fix openssh client install

### PR DESCRIPTION
**Pull request by chmouel**
openssh-clients is called with a S

(it is without s only on debianies distros)

/cc @hrishin @vdemeester

### CODE REVIEW

File: openshift/ci-operator/Dockerfile-git.in [hunk 1]
```diff
@@ -4,7 +4,7 @@ FROM registry.access.redhat.com/ubi8/ubi:latest
 # NOTE(chmou): We use dollar here so that envsubst don't get confused and expand
 # our local PATH.
 ENV HOME=/ko-app PATH=DOLLAR{HOME}:DOLLAR{PATH}
-RUN yum install --disableplugin=subscription-manager -y git openssh-client
+RUN yum install --disableplugin=subscription-manager -y git openssh-clients
```
Comment (hrishin) [inline openshift/ci-operator/Dockerfile-git.in]: :rocket:

**Comment by chmouel** [comment]
let's get this fix merged https://github.com/tektoncd/pipeline/pull/1233

**Comment by chmouel** [comment]
It's working, pushing the fix to openshift-master

**Pull request closed without merging**